tutorials/tts/Audio_Codec_Training.ipynb

```diff
@@ -0,0 +1,667 @@
+      },
+      "source": [
+        "We should listen to a few audio files before and after the processing so be sure we configured it correctly.\n",
+        "\n",
+        "Note that the processed audio is shorter because we trimmed the leading and trailing silence."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "_fM3GwJxkjOA"
+      },
+      "outputs": [],
+      "source": [
+        "audio_file = \"p228_009.wav\"\n",
+        "audio_filepath = input_audio_dir / audio_file\n",
+        "processed_audio_filepath = output_audio_dir / audio_file.replace(\".wav\", \".flac\")\n",
+        "\n",
+        "print(\"Original audio.\")\n",
+        "ipd.display(ipd.Audio(audio_filepath))\n",
+        "\n",
+        "print(\"Processed audio.\")\n",
+        "ipd.display(ipd.Audio(processed_audio_filepath))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oRO842MUyODC"
+      },
+      "source": [
+        "# Audio Codec Training"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "E4wUKYOfH8ax"
+      },
+      "source": [
+        "Here we show how to train an audio codec model from scratch. Instructions and checkpoints for fine-tuning will be provided later.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "pqfl9jAYMJob"
+      },
+      "outputs": [],
+      "source": [
+        "import torch\n",
+        "from omegaconf import OmegaConf"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "jK2rr-Kr6Qg8"
+      },
+      "outputs": [],
+      "source": [
+        "dataset_name = \"vctk\"\n",
+        "audio_dir = DATA_DIR / \"audio_preprocessed\"\n",
+        "train_manifest_filepath = DATA_DIR / \"train_manifest.json\"\n",
+        "dev_manifest_filepath = DATA_DIR / \"dev_manifest.json\""
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Vr4D-NB-yQx8"
+      },
+      "outputs": [],
+      "source": [
+        "audio_codec_training_script = NEMO_EXAMPLES_DIR / \"audio_codec.py\"\n",
+        "\n",
+        "# The total number of training steps will be (epochs * steps_per_epoch)\n",
+        "epochs = 10\n",
+        "steps_per_epoch = 10\n",
+        "\n",
+        "# Config files specifying all codec parameters\n",
+        "codec_config_dir = NEMO_CONFIG_DIR / \"audio_codec\"\n",
+        "\n",
+        "# Select model config depending on target sample rate.\n",
+        "if SAMPLE_RATE == 16000:\n",
+        "  codec_config_filename = \"audio_codec_16000.yaml\"\n",
+        "  ngc_model_name = \"audio_codec_16khz_small\"\n",
+        "  ngc_model_url = \"https://api.ngc.nvidia.com/v2/models/nvidia/nemo/audio_codec_16khz_small/versions/v1/files/audio_codec_16khz_small.nemo\"\n",
```

Comment:
side note: we can also get list of NGC pretrained models from model class, to avoid hardcoding here.